core/licenses/jna-4.4.0.jar.sha1

```diff
@@ -1 +1 @@
-6edc9b4514969d768039acf43f04210b15658cd7
\ No newline at end of file
+d85e9ceee4d97e738524a231687ff28446376c97
```

Comment:
The SHA changes because there are more bits (the aarch64-related files). The version doesn't change because it's apparently tied to the upstream JNA version, which is still `4.4.0` AFAICT. If we can bump the version that would definitely be preferred.